_pages/docs/next-ls/editors.md

```diff
@@ -101,6 +101,32 @@ Initialization options can be configured:
      ("nextls" "--stdio=true" :initializationOptions (:experimental (:completions (:enable t)))))))
 ```
 
+### Emacs with lsp-mode
+
+[lsp-mode](https://github.com/emacs-lsp/lsp-mode)
+
+1. Install `nextls` somewhere on your `PATH`
+2. Configure `lsp-mode`
+
+```
+(use-package lsp-mode
+  :ensure t
+  :hook (elixir-ts-mode . lsp)
+  :commands lsp)
+```
+
+3. Add `nextls` to the `lsp-mode` configuration
+
+```
+		(with-eval-after-load 'lsp-mode
+		  (lsp-register-client
+		   (make-lsp-client :new-connection (lsp-stdio-connection '("nextls" "--stdio" ))
+							:multi-root t
+							;; :initialization-options '(:experimental (:completions (:enable t))) ;; Enable the experimental completion mode
+							:activation-fn (lsp-activate-on "elixir")
+							:server-id 'next-ls)))
+```
```

Comment:
```suggestion
```elisp
(with-eval-after-load 'lsp-mode
  (lsp-register-client
   (make-lsp-client :new-connection (lsp-stdio-connection '("nextls" "--stdio" ))
          :multi-root t
          ;; :initialization-options '(:experimental (:completions (:enable t))) ;; Enable the experimental completion mode
          :activation-fn (lsp-activate-on "elixir")
          :server-id 'next-ls)))
```
```